ck.sh

```diff
@@ -85,6 +85,53 @@ case "$COMMAND" in
 		esac
 		;;
 
+	buildskin)
+		cd skins
+
+		echo "Which skin do you want to build?"
+
+		select skin in *
+		do
+			echo "you selected $skin"
```

Comment:
AFAIK, if I select a bogus entry, `$skin` will be an unset or an empty string which means that we'll ed up running:

```
java -jar dev/builder/ckbuilder/2.3.2/ckbuilder.jar --build-skin ../skins/ ../ckeditor/skins/
```

So we probably need a check for valid input before proceeding.

Reply:
Roger that

Reply:
but here we're selecting a skin from a list, how a bogus entry could be selected?

Reply:
said nothing

Reply:
You can type random crap in, as you've probably discovered.

Reply:
yep xD